config/config-network.yaml

```diff
@@ -53,4 +58,4 @@ data:
   # For more information, visit
   # https://istio.io/docs/tasks/traffic-management/egress/
   #
-  istio.sidecar.includeOutboundIPRanges: "*"
+  # istio.sidecar.includeOutboundIPRanges: "*"
```

Comment:
Does commenting these out result in them not being parsed in unit tests?  I worry that these get stale pretty soon.

Reply:
Another (usability?) problem here is that when people does `kubectl edit` they probably won't see these commented out values?

I was wondering if we can prefix these with something "special" (hack?) like underscore so they still show when user does `kubectl edit cm`.  If we do that we can also remove the defaulting logic that scatter through the code.  What do you think?

Reply:
I opened https://github.com/knative/serving/issues/3066 to track the idea we discussed offline.